exercises/practice/hamming/.docs/instructions.append.md

```diff
@@ -0,0 +1,14 @@
+# Implementation notes: exceptions
+
+This exercise expects the code to raise exceptions in certain circumstances.
+Exceptions are usually raised when an operation cannot be completed for some
```

Comment:
FWIW, general markdown style on Exercism is once sentence per line, not limited to a width. This makes reviews, edits, and diffs easier!